registration/views.py

```diff
@@ -1,12 +1,39 @@
 from django.shortcuts import render, render_to_response
-from registration.forms import ScoutForm
+from django.http import HttpResponseRedirect
+from django.core.context_processors import csrf
+from django.contrib.auth.models import User
+from scout.models import Scout
+from django.contrib import messages
+from registration.forms import ScoutFormSet, ScoutmasterFormSet, MbuUserCreationForm
 
 # Create your views here.
 
 def register_scout(request):
-    args = {}
-    args.update({'form': ScoutForm() })    
-    return render_to_response('registration/register_scout.html', args)
+    args = {'title': 'Register Scout'}
+    FormSet = ScoutFormSet
+    return _register(request, FormSet, args)
 
 def register_scoutmaster(request):
-    pass
+    args = {'title': 'Register Scoutmaster'}
+    FormSet = ScoutmasterFormSet
+    return _register(request, FormSet, args)
+
+def _register(request, FormSet, args):
```

Comment:
Abstraction Addison and I put together -- does all the work based on what formset object you pass in.